<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindow</class>
 <widget class="QWidget" name="HomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>动脑单车</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image/image.qrc">
    <normaloff>:/home/home_title_icon.png</normaloff>:/home/home_title_icon.png</iconset>
  </property>
  <widget class="QPushButton" name="user_center_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>54</width>
     <height>60</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>55</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/home/home_user_center_icon.png</normaloff>
     <normalon>:/home/home_user_center_icon.png</normalon>:/home/home_user_center_icon.png</iconset>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>550</width>
     <height>710</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="tipLoginWidget" native="true">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>520</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 144, 46);</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>501</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>你尚未完成手机验证，请先进行手机验证。</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="requestLoginButton">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
color: rgb(255, 85, 0);
font: 10pt &quot;新宋体&quot;;
font: 75 9pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>182</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QWidget" name="scanWidget" native="true">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>580</y>
       <width>300</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(39, 39, 39);</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>301</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="cameraScanButton">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="scanTipLabel">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
         </property>
         <property name="text">
          <string>扫码开锁</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>551</width>
       <height>711</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="mapLayout">
      <item>
       <widget class="QWidget" name="widget_2" native="true"/>
      </item>
     </layout>
    </widget>
    <zorder>label_3</zorder>
    <zorder>verticalLayoutWidget</zorder>
    <zorder>tipLoginWidget</zorder>
    <zorder>scanWidget</zorder>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QPushButton" name="bikeButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>动脑单车</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="messages_button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>45</width>
     <height>35</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/home/home_messages_icon.png</normaloff>:/home/home_messages_icon.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="locationButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>675</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="ridingInfowidget" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>240</y>
     <width>520</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ridingInfoLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="moneyLabel">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>价格:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeLabel">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tripLabel">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>里程:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishRideButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;黑体&quot;;
background-color: rgb(255, 255, 127);</string>
       </property>
       <property name="text">
        <string>停止骑行</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
    <zorder>tripLabel</zorder>
    <zorder>timeLabel</zorder>
    <zorder>moneyLabel</zorder>
    <zorder>finishRideButton</zorder>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>user_center_button</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>bikeButton</zorder>
  <zorder>messages_button</zorder>
  <zorder>locationButton</zorder>
  <zorder>ridingInfowidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image/image.qrc"/>
 </resources>
 <connections/>
</ui>
